src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -330,9 +330,13 @@ const ProjectSelector = createReactClass({
           <Link to={`/${org.slug}/`} className="home-crumb">
             <span className="icon-home" />
           </Link>
-          {this.state.activeProject
-            ? this.getLinkNode(this.state.activeTeam, this.state.activeProject)
-            : t('Select a project')}
+          {this.state.activeProject ? (
+            this.getLinkNode(this.state.activeTeam, this.state.activeProject)
+          ) : (
+            <span onClick={() => (this.state.isOpen ? this.onClose() : this.onOpen())}>
```

Comment:
We should use an unstyled Button (does that exist?) - or at least make sure it has `role="button"` and `cursor: pointer`

Reply:
good catch, will do